src/main/java/at/porscheinformatik/sonarqube/licensecheck/npm/PackageJsonDependencyScanner.java

```diff
@@ -78,7 +79,7 @@ private Set<Dependency> dependencyParser(File baseDir, InputFile packageJsonFile
                 });
             }
         }
-        catch (IOException e)
+        catch (Exception e)
```

Comment:
javax.json.stream.JsonParsingException was not caught here because it is not a subclass of IOException